adapter/network_management.md

```diff
@@ -43,8 +43,10 @@ Every adapter should define its own service name.
 | in   | cmd.network.node_update      | string¹    | Requests an interview and discovery of its capabilities for the provided address of a thing.                      |
 | in   | cmd.network.reset            | null       | Requests forceful removal of all things and resets the network.                                                   |
 | out  | evt.network.reset_done       | null       | Reports that network reset has been completed.                                                                    |
+| in   | cmd.ping.send                | string     | Requests execution of a ping for the provided address of a thing.                                                 |
```

Comment:
adress is a string ? Havent you changed it recently from string to int somwhere?

Reply:
Nope, from int to string.You got confused.